### PR TITLE
fix tests and TravisCI environment

### PR DESCRIPTION
**Pull request by andreiglingeanu**
This is a split of the #117. For the details about what this pull request does look there.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+bower_components/
```
Comment (kottenator) [inline .gitignore]: Please, add `/` at the beginning

File: .travis.yml [hunk 2]
```diff
@@ -1,11 +1,17 @@
 language: node_js
 node_js:
-  - 0.10
+  - 5.0.0
```
Comment (kottenator) [inline .travis.yml]: Why 5, not 6?

File: .travis.yml [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
 before_script:
   - npm install -g bower karma-cli
   - bower install
+  - cd bower_components/modernizr
```
Comment (kottenator) [inline .travis.yml]: I'd totally get rid of Bower in the future, but that can be done separately.

File: .travis.yml [hunk 4]
```diff
@@ -1,11 +1,17 @@
...
+  - npm install
+  - ./bin/modernizr -c lib/config-all.json
+  - cd ../..
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+addons:
+  firefox: latest
```
Comment (kottenator) [inline .travis.yml]: Is it really necessary? It's very strange, circles should work in old Firefox.

File: tests/tests.js [hunk 5]
```diff
@@ -148,7 +148,7 @@
             assert.expect(9);
             image.src = imageUrl;
 
-            $(image).load(function() {
+            $(image).ready(function() {
```
Comment (kottenator) [inline tests/tests.js]: According to [jQuery docs](https://api.jquery.com/ready/), `.ready()` is for document loading. How does it work with image?

File: .travis.yml (hunk 4 above)
Comment (andreiglingeanu) [inline .travis.yml]: Yes, it is. It's necessarily that it doesn't work in old Firefox, it's just something that stops us from passing tests in there. Maybe they don't you allow to set built-in variables on `window` there?

File: .travis.yml (hunk 3 above)
Comment (andreiglingeanu) [inline .travis.yml]: Then you'll have another way of including modernizr till then.

File: .travis.yml (hunk 2 above)
Comment (andreiglingeanu) [inline .travis.yml]: Don't know, nothing special.

File: tests/tests.js (hunk 5 above)
Comment (kottenator) [inline tests/tests.js]: Without this change, does it break the tests?

File: .travis.yml (hunk 4 above)
Comment (kottenator) [inline .travis.yml]: OK

File: tests/tests.js (hunk 5 above)
Comment (andreiglingeanu) [inline tests/tests.js]: Yes, it breaks the tests - that's [how](https://github.com/kottenator/jquery-circle-progress/issues/112).

I'm pretty sure here `$(image).ready` is similar to `$(document).ready` and it is executed not at the same time as the images are `load`-ed. It kinda makes some sense, but I'm still not sure why it fix the problem.

**Comment by andreiglingeanu** [comment]
@kottenator phantomjs didn't started in the last build. Could you please schedule one more?

**Comment by kottenator** [comment]
@andreiglingeanu: CI build restarted.

**Comment by andreiglingeanu** [comment]
@kottenator There's something weird going on after moving to `node 6.0.0`.

**Comment by andreiglingeanu** [comment]
Yep, this way we are okay.

**Comment by kottenator** [comment]
I'm merging this PR. However, I've created #119 for future improvements.

**Pull request merged**